Mage.Tests/src/test/java/org/mage/test/cards/targets/attacking/ReselectDefenderTargetEffectTest.java

```diff
@@ -0,0 +1,74 @@
+package org.mage.test.cards.targets.attacking;
+
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestCommander4Players;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import mage.counters.CounterType;
+
+public class ReselectDefenderTargetEffectTest extends CardTestCommander4Players {
```

Comment:
can we also add a test case for declare attacking planeswalker, reselect to attack player? maybe with [[Blazing Archon]] to check the ignoring of restrictions?